standard/flags.lua

```diff
@@ -160,7 +160,10 @@ function Flags.CountryName(flagName)
 		return MasterData.data[flagKey].name
 	else
 		mw.log('Unknown flag: ', flagName)
-		return mw.text.trim(mw.text.split(Template.safeExpand(mw.getCurrentFrame(), 'Flag/' .. flagName), '|', true)[2] or '')
+		return mw.text.trim(mw.text.split(mw.text.split(
```

Comment:
lets set `Category:Pages with unknown countries` here
(since we can now set it directly from lua instead of having to have it as text in output it should be easy now^^